README.md

```diff
@@ -2,14 +2,69 @@
+Currently a number of ICOs are already available and more can be manually added to the Transparency-Monitor in order study a specific ICO or compare between different ICOs.
+## How it works
+
+### Data Collection
+All [transactions](http://solidity.readthedocs.io/en/develop/introduction-to-smart-contracts.html#index-8) executed in the ethereum network are logged and stored as [blocks](http://solidity.readthedocs.io/en/develop/introduction-to-smart-contracts.html#blocks) in the public ethereum blockchain. These transactions can either be a transfer or execution of code in the form of a [Smart Contract](http://solidity.readthedocs.io/en/develop/introduction-to-smart-contract).<br/>
+
+ When a Smart Contract is executed, it generates  [log](http://solidity.readthedocs.io/en/develop/introduction-to-smart-contracts.html#logs) events based on how it is programmed and what function is  executed at the time. In most cases ICOs are token generating smart contracts, where every token generation by the ICO's smart contract is permanently logged in the blockchain.
+
+ Using [Web3](https://github.com/ethereum/wiki/wiki/JavaScript-API#watch-callback-return-value) The ICO-Monitor takes the address of an ICO Smart Contract and scans through the whole blockchain and collects all token generation events logged by the ICO smart contract.
+
+### Questions
+In addition to the collected data, ICO smart contracts are assigned a class based on a decision matrix by manually answering these questions:
+
+1. Is ICO controlled by a smart contract? (some ICOs are entirely performed in the backend and smart contract is created post factum. Example ICONOMI
+
+2. Is smart contract source code available? (license type ie. if this is open source or not does not matter)
+
+3. Is smart contract source code provided in etherscan?
+
+4. Is instruction provided how to reproduce deployed byte-code? (does not apply if etherscan source is there)
+
+5. Does smart contract provide all tracking data via events? (if it is easy to read accounts, tokens, and token price from events)
+
+6. Is information on token price in ETH provided? (via event or in transaction?)
+
+7. Does smart contract handle ETH in a trustless way? Is ETH really sent to ICO smart contract in a transaction or we need to trust some backend on it. Please note that there can be ICO without ether at all (like Neufund ICO)
+
+8. If ICO is using other currencies is information on token price provided?
+
+9. Does smart contract handle other currencies in a trustless way? Does some smart contract store balance of those currencies? (it is like our EUR Token)
+
+10. Was smart contract code easy to read and properly commented?
+11. the ICO doing exactly the same what they say on their website ?
+
+### Classes
+Using the collected data and answered questions an ico is given one of these states:
```

Comment:
" .. questions an ico is given .."
ico should be written in capital letters "ICO"